gui/src-tauri/src/main.rs

```diff
@@ -258,7 +280,9 @@ fn main() {
 			}
 			return;
 		}
-		_ => run_result.expect("error while running tauri application"),
+		Err(_) => {
+			println!("Unknown Error")
```

Comment:
This should be a tauri message dialog, right? println is not useful for an app that is double clicked like this.

Reply:
changed